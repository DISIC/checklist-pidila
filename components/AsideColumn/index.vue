<template>
  <div class="col-aside">
    <aside>
      <slot/>
    </aside>
  </div>
</template>

<script>
  export default {
    name: 'AsideColumn'
  }
</script>
