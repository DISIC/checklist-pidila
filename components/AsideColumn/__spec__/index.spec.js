import {shallowMount} from '@vue/test-utils'

import AsideColumn from '../index.vue'

describe('components | AsideColumn', () => {

  it('should match snapshot', () => {
    // when
    const wrapper = shallowMount(AsideColumn)

    // then
    expect(wrapper.element).toMatchSnapshot()
  })

})
