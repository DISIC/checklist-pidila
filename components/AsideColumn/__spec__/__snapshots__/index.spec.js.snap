// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | AsideColumn should match snapshot 1`] = `
<div
  class="col-aside"
>
  <aside />
</div>
`;
